airflow/api/common/experimental/mark_tasks.py

```diff
@@ -33,6 +34,7 @@
 from airflow.utils.types import DagRunType
 
 
+@deprecated(reason="Use airflow.api.common.mark_tasks._create_dagruns instead", version="2.2.3")
 def _create_dagruns(dag, execution_dates, state, run_type):
```

Comment:
This is a private function (that's the convention in Python for a leading underscore) so we shouldn't have to deprecate this one, we can just delete it and update what ever uses it to use the new version.